src/main/java/org/broadinstitute/hellbender/utils/io/Resource.java

```diff
@@ -84,13 +84,16 @@ public InputStream getResourceContentsAsStream() {
      */
     public static File getResourceContentsAsFile(final String resourcePath) throws IOException {
         final File tmpResourceFile = File.createTempFile("tmp_read_resource_", ".config");
-        final InputStream systemResourceAsStream = ClassLoader.getSystemResourceAsStream(resourcePath);
+        InputStream resourceAsStream = ClassLoader.getSystemResourceAsStream(resourcePath);
 
-        if (systemResourceAsStream == null) {
-            throw new GATKException("Null value when trying to read system resource.  Cannot find: " + resourcePath);
+        if (resourceAsStream == null) {
+            resourceAsStream = Resource.class.getClassLoader().getResourceAsStream(resourcePath);
```

Comment:
Can't we just use this statement above and eliminate the `if` statement?

I.e. replace `InputStream resourceAsStream = ClassLoader.getSystemResourceAsStream(resourcePath);` with 
`InputStream resourceAsStream = Resource.class.getClassLoader().getResourceAsStream(resourcePath);`

I would think that the `ClassLoader` would be equivalent to `Resource.class.getClassLoader()` 

Apologies if I am forgetting something....

Reply:
I think you're right. A test seems to be failing now, but I'm not sure it's related. I'll investigate.